src/test/resources/data/typicalDataBundle.json

```diff
@@ -269,6 +269,17 @@
             "isDisplayedToStudents":true,
             "displayedName":"Instructor",
             "instructorPrivilegesAsText": "{\n  \"courseLevel\": {\n    \"canviewstudentinsection\": true,\n    \"cangivecommentinsection\": true,\n    \"cansubmitsessioninsection\": true,\n    \"canmodifysessioncommentinsection\": true,\n    \"canmodifycommentinsection\": true,\n    \"canmodifycourse\": true,\n    \"canviewsessioninsection\": true,\n    \"canmodifysession\": true,\n    \"canviewcommentinsection\": true,\n    \"canmodifystudent\": true,\n    \"canmodifyinstructor\": true\n  },\n  \"sectionLevel\": {},\n  \"sessionLevel\": {}\n}"
+        },
+        "instructorNotYetJoinCourse1":{
+            "googleId":null,
+            "courseId":"idOfTypicalCourse1",
```

Comment:
Can you move this nearer to the top, where the other instructors for the course idOfTypicalCourse1 are? The rest of the instructors are not very neatly organised, but I guess we shouldn't mess things up more. 

Have you tried running the other tests? Many tests use the typicalDataBundle and modifications to this file may affect the other tests.